test/codeu/chat/server/DatabaseTest.java

```diff
@@ -0,0 +1,29 @@
+package codeu.chat.server;
+
+import static org.junit.Assert.assertTrue;
+
+import java.util.UUID;
+import org.junit.Test;
+
+
+public final class DatabaseTest {
+
+  private static final DBConnector.Connector con = new DBConnector.Connector();
```

Comment:
`@Before` with `setUp()` and `@After` with `tearDown()` would probably be useful with your connector.